server/topic.go

```diff
@@ -782,39 +784,12 @@ func (t *Topic) handleSubscription(h *Hub, sreg *sessionJoin) error {
 	return nil
 }
 
-// Send presence notification in response to a subscription
-func (t *Topic) sendSubNotifications(asUid types.Uid, sreg *sessionJoin) {
+// Send immediate presence notification in response to a subscription.
+// These notifications are always sent immediately even if background is requested.
```

Comment:
This defeats the purpose of introducing background requests.
We don't care about presence notifications when background a request is sent. Should we discuss this further?

Reply:
I split the notifications into two sets: sent immediately regardless of the background status and those optionally delayed. The always immediate notifications are p2p & group notifications on a new subscription. I think it's a weird case to join a new chat in background.

Reply:
Okay, thanks for explaining.
This is good.